python/tvm/relay/frontend/paddlepaddle.py

```diff
@@ -514,6 +582,27 @@ def convert_expand_as(g, op, block):
     g.add_node(op.output("Out")[0], out)
 
 
+def convert_eye(g, op, block):
+    """Operator converter for eye."""
+
+    num_rows = op.attr("num_rows")
+    num_columns = op.attr("num_columns")
```

Comment:
num_columns might equal -1, in which case num_columns equals num_rows.

Reply:
Refer to the code https://github.com/PaddlePaddle/Paddle/blob/release/2.4/python/paddle/tensor/creation.py#L808, num_columns should be a non-negative int

Reply:
We should look at the c++ definition and implementation of op rather than the api.
The Paddle to TVM conversion is an op mapping rather than an api.
https://github.com/PaddlePaddle/Paddle/blob/b780a3ff4f89f73f4efd095002b320a5fe673afe/paddle/phi/kernels/impl/eye_kernel_impl.h#L44